internal/nodes/avi_model_evh_nodes.go

```diff
@@ -2099,6 +2099,12 @@ func RouteIngrDeletePoolsByHostnameForEvh(routeIgrObj RouteIngressModel, namespa
 		}
 
 		SharedHostNameLister().DeleteNamespace(host)
+		if found, ingressHostMap := SharedHostNameLister().Get(host); found {
```

Comment:
@aaha97 Similar problem should exists for SNI insecure ingress deletion. Did we face issue over there?

Reply:
this was found when running evhtests. might find the issue in SNI UTs.

Reply:
added the fix for SNI case